test/unit/uk/gov/hmrc/thirdpartyapplication/controllers/AuthorisationWrapperSpec.scala

```diff
@@ -183,21 +177,21 @@ class AuthorisationWrapperSpec extends UnitSpec with MockitoSugar with WithFakeA
 
     "accept the request when the gatekeeper is logged in" in new Setup {
 
-      mockAuthConnectorToReturn(true)
+      givenUserIsAuthenticated(underTest)
 
-      val result = await(underTest.requiresRole(APIGatekeeper).async(_ => Default.Ok(""))(FakeRequest()))
+      val result = await(underTest.requiresRole().async(_ => Default.Ok(""))(FakeRequest()))
 
       status(result) shouldBe SC_OK
     }
 
-    "return a 401 (Unauthorised) response when the gatekeeper is not logged in" in new Setup {
+    "return a 403 (forbidden) response when the gatekeeper is not logged in" in new Setup {
 
-      mockAuthConnectorToReturn(false)
+      givenUserIsNotAuthenticated(underTest)
 
-      val result = await(underTest.requiresRole(APIGatekeeper).async(_ => Default.Ok(""))(FakeRequest()))
+      val result = await(underTest.requiresRole().async(_ => Default.Ok(""))(FakeRequest()))
 
-      status(result) shouldBe SC_UNAUTHORIZED
-      jsonBodyOf(result) shouldBe JsErrorResponse(UNAUTHORIZED, "Action requires authority: 'api:gatekeeper'")
+      status(result) shouldBe SC_FORBIDDEN
+      jsonBodyOf(result) shouldBe JsErrorResponse(FORBIDDEN, "Insufficient enrolments")
     }
   }
```

Comment:
I think this whole spec needs a rethink. It strikes me that maybe the behaviour being tested and the language being used no longer match the requirements.